documentation/src/object-Initializer.md

```diff
@@ -0,0 +1,14 @@
+# Initializer
```

Comment:
Something's wrong with the casing of the file name. I think it should be kebab-case like all the other files.

Reply:
Ooops, that was a stupid mistake...

Reply:
a307d749dc25dd00af008083a96e8e6b9891d827